src/coreclr/nativeaot/System.Private.CoreLib/src/CompatibilitySuppressions.xml

```diff
@@ -797,6 +797,10 @@
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Diagnostics.DebugAnnotations</Target>
   </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Diagnostics.DebuggerGuidedStepThroughAttribute</Target>
```

Comment:
Yep, now we know why these were flagged before https://github.com/dotnet/runtime/pull/89216

No need to file an issue here, since it seems we had previously suppressed them.